pages/common/spectacle.md

```diff
@@ -0,0 +1,16 @@
+# spectacle
+
+> KDE's screenshot utility.
+> More information: <https://manned.org/spectacle>.
```

Comment:
I think the author's web is better
```suggestion
> More information: <https://apps.kde.org/spectacle/>.
```

Reply:
Generally, We can use KDE docs for most commands. But there is no documentation on flags in https://docs.kde.org/stable5/en/spectacle/spectacle/index.html or the link you suggested in this case, I think we can either link to manned or to docs but it doesn't have any info on command line arguments.

Reply:
I like the man page from manned.org more.

Reply:
- I think that we should always use the "closest" page to written examples: the page which has all used flags mentioned in examples at the bare minimum level and maybe even more.
- Also I suggest check whether such page is actively supported: prefer the page with the most recent changes.

Reply:
I guess this conversation can be marked as resolved as PR was approved.